### PR TITLE
Math magicians: Components

### PR DESCRIPTION
**Pull request by lily-coder**
In this section, I have continued with the development of the Math Magicians app. I have developed just the UI of the calculator

### CODE REVIEW

**Review by wandji20 — CHANGES_REQUESTED** [review]
## Hi @lily-coder :raised_hands:

# Status: Invalid :stop_sign:

> Friendly reminder: You should make sure that your repo looks professional before review.<br>It will be nice to use this [template](https://github.com/microverseinc/readme-template) for your README.
This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

**Good luck :four_leaf_clover:** 

 If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily-coder** [comment]
Thank you @wandji20. I have made the necessary changes

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,13 @@
-import logo from './logo.svg';
+/* eslint no-unused-vars: 0 no-undef: 0 */
+import React from 'react';
 import './App.css';
+import Calculator from './components/Calculator.js';
 
 function App() {
   return (
-    <div className="App">
```
Comment (henatan99) [inline src/App.js]: - [x] The App component should be class-based as per the project's requirement. In this milestone, the objective is to practice react class-based components. Kindly refactor this component to a class-based component. :pray:

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/* eslint no-unused-vars: 0 no-undef: 0 */
+import React from 'react';
+import '../App.css';
+
+export default function Calculator() {
+  return (
+    <div class='calculator-container'>
```
Comment (henatan99) [inline src/components/Calculator.js]: - [x] The Calculator.js component should be class-based as per the project's requirement. In this milestone, the objective is to practice react class-based components. Kindly refactor this component to a class-based component. :pray:

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      <div>
+        <button class='calc-row-one'>0</button>
```
Comment (henatan99) [inline src/components/Calculator.js]: - [x] I don't think `button` tag is appropriate here. Kindly use a more semantically correct tag instead. :pray:

File: src/App.css [hunk 4]
```diff
@@ -1,39 +1,60 @@
...
+  background-color: rgba(128, 128, 128, 0.712);
+}
+
+.calc-row-two button,
+.calc-row-three button,
+.calc-row-four button,
+.calc-row-five button,
+.calc-row-six button {
+  width: 6rem;
 }
```
Comment (henatan99) [inline src/App.css]: - [x] Kindly make your styling represent the original design. I think you need to change the positioning and color of text inside the display field. 

image
image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @lily-coder , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily-coder** [comment]
Thank you @henatan99

**Review by barackm — APPROVED** [review]
Hi @lily-coder, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily-coder** [comment]
Thank you @barackm

**Pull request merged**